pgo/cmd/restore.go

```diff
@@ -30,7 +30,7 @@ import (
 )
 
 var PITRTarget string
-var BackupPath, BackupPVC string
+var BackupPath, BackupPVC, PGDumpRestoreDB string
```

Comment:
We could probably reuse `PGDumpDB` (or something like `Database` or `DatabaseName`) given there's no overlap. I'm not a fan of that over the long term (we likely need to reorganize how all of these variables are stored).

Reply:
Sure thing. I'll plan to reorganize this so that PGDumpDB is reused in both places, with a comment indicating it's being used that way. I'll look at the request message structures as well while I'm there. Thanks!